openedx_learning/apps/authoring/publishing/api.py

```diff
@@ -1018,3 +1087,25 @@ def get_containers_with_entity(
     #     publishable_entity__draft__version__containerversion__entity_list__in=lists
     # )
     return qs
+
+
+def get_container_children_count(
+    container: Container,
```

Comment:
The other container API methods take a `container_key` instead of a `Container` object -- can this one too?

Reply:
All container API dealing with entities/children under it expect a `Container` object:
* [Ref 1](https://github.com/open-craft/openedx-learning/blob/7552d7a762707110d255034bd4fcd6c042f3c16d/openedx_learning/apps/authoring/publishing/api.py#L972-L973)
* [Ref 2](https://github.com/open-craft/openedx-learning/blob/7552d7a762707110d255034bd4fcd6c042f3c16d/openedx_learning/apps/authoring/units/api.py#L245)
* [Ref 3](https://github.com/open-craft/openedx-learning/blob/7552d7a762707110d255034bd4fcd6c042f3c16d/openedx_learning/apps/authoring/units/api.py#L270)

Reply:
Ah ok.. I guess it's fine as is. Will continue my review!